### PR TITLE
#745 used projectManagerID and projectLeadId to store numbers

### LINKED ISSUE #745: WP Edit - change name of project lead and manager fields
### Description

Right now the fields sent to the work package edit endpoint are `projectLead` and `projectManager` which is confusing because what the endpoint expects is the id

### Acceptance Criteria

make the route use `projectLeadId` and `projectManagerId` instead

### Proposed Solution

this will involve a bunch of changes:

- change it in work-packages.routes.ts
- change the value that gets extracted in the work packages controller file
- change the name of the arguments in the work packages service file
- change the payload created in line `106` of `WorkPackageEditContainer.tsx`

### PR DESCRIPTION
**Pull request by jakewc12**
## Changes

Changed WorkPackageEditContainer.tsx and work-packages.routes.ts to incorporate projectManagerId and projectLeadId instead of projectManager and projectLead

## Notes

The instructions said to change work-packages.controller.tsx as well but I'm not sure if it should be changed. Please let me know!

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ x] All commits are tagged with the ticket number
- [ x] No linting errors / newline at end of file warnings
- [ x] All code follows repository-configured prettier formatting
- [ x] No merge conflicts
- [x ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [x ] Remove any non-applicable sections of this template
- [ x] Assign the PR to yourself
- [x ] No `yarn.lock` changes (unless dependencies have changed)
- [x ] Request reviewers & ping on Slack
- [ x] PR is linked to the ticket (fill in the closes line below)

Closes #745

### CODE REVIEW

**Review by walker-sean — CHANGES_REQUESTED** [review]
Nice job so far! You are also going to have to change the fields in the work package controller and work package services files. After that, you should be good!

Also, can you please put your ticket number in the `Closes #` section of the PR. Thanks!

File: src/backend/src/services/work-packages.services.ts [hunk 1]
```diff
@@ -447,8 +447,8 @@ export default class WorkPackagesService {
         wbsElement: {
           update: {
             name,
-            projectLeadId: projectLead,
-            projectManagerId: projectManager
+            projectLeadId: projectLeadId,
```
Comment (walker-sean) [inline src/backend/src/services/work-packages.services.ts:450]: you can make this line simply `projectLeadId,`

File: src/backend/src/services/work-packages.services.ts [hunk 2]
```diff
@@ -447,8 +447,8 @@ export default class WorkPackagesService {
...
+            projectManagerId: projectManagerId
```
Comment (walker-sean) [inline src/backend/src/services/work-packages.services.ts:451]: you can make this line simply `projectManagerId`

**Review by walker-sean — CHANGES_REQUESTED** [review]
Looks really good!! Just one more thing – to fix your linting check, you want to use a shorthand that lets you just type the name of the argument by itself (as seen in my comments)

File: src/backend/src/services/work-packages.services.ts (hunk 2 above)
Comment (jakewc12) [reply src/backend/src/services/work-packages.services.ts]: I think I did these now. Lmk if I have to change them or not.

**Review by walker-sean — DISMISSED** [review]
LGTM

File: src/backend/src/controllers/work-packages.controllers.ts [hunk 3]
```diff
@@ -73,7 +73,7 @@ export default class WorkPackagesController {
         expectedActivities,
         deliverables,
         projectLead,
```
Comment (RChandler234) [inline src/backend/src/controllers/work-packages.controllers.ts:75]: need to update to projectLeadId as well

File: src/backend/src/services/work-packages.services.ts [hunk 4]
```diff
@@ -281,8 +281,8 @@ export default class WorkPackagesService {
     blockedBy: WbsNumber[],
     expectedActivities: DescriptionBullet[],
     deliverables: DescriptionBullet[],
-    projectLead: number,
-    projectManager: number
+    projectLeadId: number,
```
Comment (RChandler234) [inline src/backend/src/services/work-packages.services.ts:284]: can you update the comment for this service method to use the right names as well

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**